ComcastCRMGUIFramework/src/main/java/com/comcast/crm/objectrepositoyUtility/CreateProductPage.java

```diff
@@ -0,0 +1,9 @@
+package com.comcast.crm.objectrepositoyUtility;
+
+public class CreateProductPage {
+
+	
```

Comment:
dont hardcode

change the xpath 
use relative xpath instead of absolute xpath

Reply:
fixed as per reviwed